site/src/pages/WorkspacePage/Workspace.tsx

```diff
@@ -257,6 +282,53 @@ export const Workspace: FC<WorkspaceProps> = ({
   );
 };
 
+const WorkspaceResourceData: FC<{ resource: TypesGen.WorkspaceResource }> = ({
+  resource,
+}) => {
+  const metadata = resource.metadata ?? [];
+
+  if (resource.daily_cost > 0) {
+    metadata.push({
+      key: "Daily cost",
+      value: resource.daily_cost.toString(),
+      sensitive: false,
+    });
+  }
+
+  if (metadata.length === 0) {
+    return null;
+  }
+
+  return (
+    <header css={styles.resourceData}>
+      {resource.metadata?.map((meta) => {
```

Comment:
Should we be mapping over `resource.metadata` here or your new `metadata` array?

Reply:
Good catch!